explainer.md

```diff
@@ -508,30 +522,31 @@ The goal of user agents should be to ensure that the PEPC is not trivial to
 abuse. Therefore the user agent should consider the following potentially
 malicious tactics and mitigate them:
 
-- The site could trick the user by choosing some misleading text (e.g. "Click
-  here to proceed"). Therefore the text on the PEPC should not be able to be set
-  by the site, instead the user agent should make sure to set it to something
-  comprehensive (e.g. "Share location" for a geolocation PEPC). \
-   **Open question:**should there be a mechanism that allows the site to pick
-  one of several flavors of text (example: "Share location" vs "Use location")?
-- The style of the PEPC can be set to obscure the purpose (e.g. setting the same
-  text color and button color would make the text unreadable). Therefore the
-  style should be verified, validated and overridden by the user agent as
-  needed. More details in the [Locking the PEPC style](#locking-the-pepc-style)
-  section
-- The PEPC might be partially covered (to hide the text) with another HTML
-  element. Therefore the user agent should verify that the PEPC has been visible
-  already for some short time (e.g. 500ms or so) before it's clicked. User
-  agents that implement the
-  [IntersectionObserverV2](https://github.com/w3c/IntersectionObserver/blob/v2/explainer.md)
-  API can make use of it internally.
-- The site might try to obtain a click on the PEPC by moving it where the user
-  is about to click. Therefore the user agent should ensure that the PEPC has
-  not been moved recently (e.g. in the past 500ms or so).
-- The site might try to obtain a click on the PEPC by inserting it into the DOM
-  where the user is about to click. Therefore the user agent should ensure that
-  the PEPC has not been inserted into the DOM recently (e.g. in the past 500ms
-  or so).
+-   The site could trick the user by choosing some misleading text (e.g. "Click
+    here to proceed"). Therefore the text on the PEPC should not be able to be
+    set by the site, instead the user agent should make sure to set it to
+    something comprehensive (e.g. "Share location" for a geolocation PEPC). \
+    **Open question:**should there be a mechanism that allows the site to pick
```

Comment:
I think this is where other solutions have broken down (I hate to say it, but the iOS one)... it's so customizable to the point that it becomes meaningless. 

And for Geolocation, for instance, sometimes you really just want a button that's as simple as: 
![Screenshot 2023-12-20 at 14 02 39](https://github.com/WICG/PEPC/assets/870154/a5766d64-24d9-44e6-b024-a6553517dcd0)

Providing native text is also super challenging, because each browser might decide to provide different text... otherwise we would need to standardize every text variation for every possible language for every possible option. 

That's going to be super challenging because if one browser chooses one word, another chooses another, and that causes a line break, then the page's design will break.

Reply:
Extracted into https://github.com/WICG/PEPC/issues/8